firefox-ios/Client/Frontend/Fakespot/Views/FakespotActionFooterView.swift

```diff
@@ -44,11 +45,10 @@ public final class ActionFooterView: UIView, ThemeApplicable {
         label.adjustsFontForContentSizeCategory = true
     }
 
-    private lazy var linkButton: LegacyResizableButton = .build { button in
+    private lazy var linkButton: ResizableButton = .build { button in
```

Comment:
This change plus actually causes UI issues where the button is either too big or displayed in with lots of space above. Let's undo this change and the removal of the buttonEdgeSpacing a few lines down and address the deprecation warning in another PR. I'll create a task for it.
![Simulator Screenshot - iPhone 15 Pro Max - 2024-02-07 at 09 35 26](https://github.com/mozilla-mobile/firefox-ios/assets/4530/7081009a-9685-499d-b176-2282ec53ae17)
![Simulator Screenshot - iPhone 15 Pro Max - 2024-02-07 at 09 35 18](https://github.com/mozilla-mobile/firefox-ios/assets/4530/a4b8e865-55a6-4fab-b610-d4e760ce52fe)

Reply:
The task to fix the deprecation warnings for the button: https://github.com/mozilla-mobile/firefox-ios/issues/18634